app/controllers/api/v2/config_reports_controller.rb

```diff
@@ -8,7 +8,7 @@ class ConfigReportsController < V2::BaseController
       before_action :setup_search_options, :only => [:index, :last]
       before_action :compatibility, :only => :create
 
-      add_smart_proxy_filters :create, :features => proc { ConfigReportImporter.authorized_smart_proxy_features }
+      add_smart_proxy_filters :create, :features => proc { Foreman::Plugin.fact_importer_registry.fact_features }
```

Comment:
Why do we use fact features to allow config reports?

Reply:
Right, I accidentally mis-read the #5010 patch I thought it was about reports while it was about facts. This is wrong, need to dig deeper.